.circleci/config.yml

```diff
@@ -272,9 +272,8 @@ jobs:
                 include_project_field: false
                 failure_message: ':red_circle: WordPress Android connected tests failed on \`${CIRCLE_BRANCH}\` branch after merge by ${CIRCLE_USERNAME}. See <https://console.firebase.google.com/u/0/project/api-project-108380595987/testlab/histories/bh.e0c3a59bd9ed670|Firebase console test results> for details.\n\nPlease reach out in #platform9 if you think this failure is not caused by your changes, so we can investigate.'
   raw-screenshots:
-    executor:
-      name: android/default
-      api-version: "28"
+    docker:
+      - image: circleci/android@sha256:061e2535826cc3fe4c4a440e716bf06c36c80401ee635c339c6803b3e427ebb3
```

Comment:
💡 Maybe for more clarity we should declare this at the top of the `.circleci/config.yml`:
```yml
executors:
  android-jdk8:
    docker:
      - image: circleci/android@sha256:061e2535826cc3fe4c4a440e716bf06c36c80401ee635c339c6803b3e427ebb3
```
And then update all the jobs needing that executor later in the config with: `executor: android-jdk8`?

Benefit would both be maintainability and readability (because now it's not clear why we're using this specific sha256 while a nice executor name or anchor name would help)

---

Alternatively, we can at least use YAML "aliases and anchors" feature to only reference that docker image name once in the config file and let the alias point to that later instead.

```yaml
test:
  docker: &android-jdk8
    - image: circleci/android@sha256:061e2535826cc3fe4c4a440e716bf06c36c80401ee635c339c6803b3e427ebb3
  steps:
    ...

...

raw-screenshots:
  docker: *android-jdk8
  steps:
    ...
```

Reply:
Good point! I'll address this in a dedicated PR. Thank you!